pkgs/by-name/bu/buildbox/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  bubblewrap,
+  makeWrapper,
+  cmake,
+  pkg-config,
+  ninja,
+  grpc,
+  gbenchmark,
+  gtest,
+  protobuf,
+  glog,
+  nlohmann_json,
+  zlib,
+  openssl,
+  libuuid,
+  tomlplusplus,
+  fuse3,
+}:
+let
+  pname = "buildbox";
+  version = "1.3.7";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  buildInputs = [
+    bubblewrap
+    fuse3
+    gbenchmark
+    glog
+    grpc
+    gtest
+    libuuid
+    nlohmann_json
+    openssl
+    protobuf
+    tomlplusplus
+    zlib
+  ];
+  nativeBuildInputs = [
+    cmake
+    makeWrapper
+    ninja
+    pkg-config
+  ];
+
+  src = fetchFromGitLab {
+    owner = "BuildGrid";
+    repo = "buildbox/buildbox";
+    tag = version;
+    hash = "sha256-US0qJrKoAYR4rMmolC8jx7IpQ2PiHZy7L2bog+I3G48=";
+  };
+
+  postFixup = ''
+    wrapProgram $out/bin/buildbox-run-bubblewrap --prefix PATH : ${lib.makeBinPath [ bubblewrap ]}
+    ln -sf buildbox-run-bubblewrap $out/bin/buildbox-run
+  '';
+
+  meta = {
```

Comment:
Missing `meta.mainProgram`

Reply:
It’s not necessary to set if there’s no sensible choice for `lib.getExe buildbox` or `nix run nixpkgs#buildbox` to use.

Reply:
Fair enough, all good here.